src/tools/cephfs/cephfs-shell

```diff
@@ -1088,25 +1088,52 @@ sub-directories, files')
         """
         self.poutput(os.getcwd())
 
+    def complete_df(self, text, line, begidx, endidx):
+        """
+        auto complete of file name.
+        """
+        return self.complete_filenames(text, line, begidx, endidx)
+
+    df_parser = argparse.ArgumentParser(description='Show information about\
+                the ceph file system on which each file resides by default.')
```

Comment:
s/ceph/Ceph

Also, I don't understand what is being meant by "on which each file resides by default".

Reply:
I have change the line as it was not appropriate.